### ISSUE TITLE
Vault doesn't return excess RUN when vaults are closed

### ISSUE THREAD

**Issue by dtribble** [Agoric/Agoric#4879]
## Describe the bug
It appears that if the vault liquidates more than the required collateral, it does not deliver the excess RUN to the vault owner.

TBD

## Expected behavior
Any RUN in excess of the debt and liquidation penalty should be provided to the vault owner on `close`.

## Platform Environment
n/a (vault.js)

## Additional context
This was found during code review.

**Comment by dtribble** [comment]
The fix appears to be to replace 
```javascript
      seat.incrementBy(
        vaultSeat.decrementBy(
          harden({ Collateral: getCollateralAllocated(vaultSeat) }),
        ),
      );
```
with
```javascript
      seat.incrementBy(vaultSeat.decrementBy(vaultSeat.getCurrentAllocation()));
```

**Comment by dtribble** [comment]
Fixed by #4877

**Comment by dtribble** [comment]
Fixed by https://github.com/Agoric/agoric-sdk/pull/4877
